ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -6,6 +6,21 @@ set -o pipefail
 
 export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
 
+if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+  # Hack to avoid importing arm64 release image by using an image override
+  # Find a better way of doing this once https://issues.redhat.com/browse/DPTP-2265 is resolved
+  touch ${SHARED_DIR}/cluster-config.yaml
+  ARM64_RELEASE_IMAGE_INITIAL=registry.ci.openshift.org/ocp-arm64/release-arm64:${BRANCH}
```

Comment:
@LorbusChris where is `BRANCH` assigned?
I see libvirt has it created but empty in: 
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/libvirt/ipi-conf-libvirt-ref.yaml#L7

Reply:
Oh. I have to check where this is populated there and so the same here